app/views/root/news.html.slim

```diff
@@ -14,7 +14,7 @@
       section.mt-6
         .container
           .row.o-card-row
-            - NewsItem.current.each_with_index do |item, i|
+            - NewsItem.current.limit(20).each_with_index do |item, i|
```

Comment:
Does this have a sort on it to ensure we always get the same 20? I'm guessing yes.

Reply:
Yes, `NewsItem.current` is sorted